pyproject.toml

```
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+exclude = ["htmlcov"] # Exclude the coverage report file from setuptools package finder
+
+[project]
+name = "WSIMOD"
+description = "WSIMOD is for simulating water quality and quantity"
+readme = "README.md"
+version = "0.2.0"
+license = {file = "LICENSE"}
+authors = [
+    { name = "Imperial College London RSE Team", email = "ict-rse-team@imperial.ac.uk" },
+    { name = "Barnaby Dobson", email = "b.dobson@imperial.ac.uk" }
+]
+requires-python = ">=3.9"
+dependencies = [
+    "PyYAML",
+    "tqdm",
+    "dill"
+]
+
+[project.optional-dependencies]
+dev = [
+    "pytest"
+]
+
+demos = [
+    "pandas",
```

I've been trying to run all the tutorials and for them to work we need to pin Pandas to 1.5.2 since they are using some features that have been removed in the current version (2.1.2). In particular, I get:

```output
  File "C:\Users\dalonsoa\Projects\wsi\docs\demo\scripts\oxford_demo.py", line 120, in <module>
    dates.sort()
    ^^^^^^^^^^
AttributeError: 'DatetimeArray' object has no attribute 'sort'
```

So we need this:

```suggestion
    "pandas==1.5.2",
```

And then recreate the `requirements-demos.txt` file. Also open an issue highlighting this, so the use of deprecated features can be fixed in the future (possibly by us).